systemtest/src/test/java/io/strimzi/systemtest/backup/ColdBackupScriptST.java

```diff
@@ -69,7 +81,17 @@ void backupAndRestore(ExtensionContext context) {
         // recreate the namespace and deploy the operator
         ResourceManager.kubeClient().deleteNamespace(NAMESPACE);
         NamespaceUtils.waitForNamespaceDeletion(NAMESPACE);
-        installClusterOperator(context, NAMESPACE);
+
+        install = new Install.InstallBuilder()
+            .withExtensionContext(context)
+            .withClusterOperatorName(Constants.STRIMZI_DEPLOYMENT_NAME)
+            .withNamespace(NAMESPACE)
+            .withWatchingNamespaces(NAMESPACE)
+            .withBindingsNamespaces(Collections.singletonList(NAMESPACE))
+            .withOperationTimeout(Constants.CO_OPERATION_TIMEOUT_DEFAULT)
+            .withReconciliationInterval(Constants.RECONCILIATION_INTERVAL)
```

Comment:
Do we really need to set them all again?

Reply:
No, I have changed the logic...when it is not set, we will set to default values.